_docs/services/app-autoscaler.md

```diff
@@ -0,0 +1,472 @@
+   - `metric_type` - One of "Scalable Metric Types" like `cpu` or `memoryused`
+   - `threshold` - The boundary when metric value exceeds is considered as a breach, always an integer value.
+   - `operator` - `>`, `<`, `>=`, `<=`
+   - `adjustment` - Describes how the number of application instances will be changed.  Support regex format `^[-+][1-9]+[0-9]*[%]?$`, i.e. +5 means adding 5 instances, -50% means shrinking to the half of current size.  
+   - `breach_duration_secs` - The time duration (integer value in seconds) to fire a scaling event if it keeps breaching. Spikes above the `threshold` that are for less than `breach_duration_secs` are thus ignored.
+   - `cool_down_secs` - The time duration (integer value in seconds) to wait before the next scaling kicks in.   
+
+
+## Recurring and Specific Schedules
+
+There is a fourth optional top level Scaling Policy key called `Schedules`:
+
+ - App-AutoScaler uses schedules to overwrite the default instance limits for specific time periods and is primarily used to prepare enough instances for peak hours.
+ - During these time periods all dynamic scaling rules are still in effect.
+ - You can define recurring schedules, or specific schedules which are executed only once.
+ - Besides overriding the default instance limit of `instance_min_count` and `instance_max_count`, you can also define an `initial_min_instance_count` in the schedule.
+
+
+
+### Recurring Schedule
+
+| Name                                 | Type                | Required| Description                                                                             |
+|:-------------------------------------|---------------------|---------|-----------------------------------------------------------------------------------------|
+| start_date                           | String,"yyyy-mm-dd" | false   | the start date of the schedule. Must be a future time.                                  |
+| end_date                             | String,"yyyy-mm-dd" | false   | the end date of the schedule. Must be a future time.                                    |
+| start_time                           | String,"hh:mm"      | true    | the start time of the schedule                                                          |
+| end_time                             | String,"hh:mm"      | true    | the end time of the schedule                                                            |
+| days_of_week / days_of_month         | Array<int>          | false   | recurring days of a week or month. Use [1,2,..,7] or [1,2,...,31] to define it          |
+| instance_min_count                   | int                 | true    | minimal number of instance count for this schedule                                      |
+| instance_max_count                   | int                 | true    | maximal number of instance count for this schedule                                      |
```

Comment:
Change minimal to minimum? Same with maximal - to maximum all throughout this table